openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx

```diff
@@ -232,7 +112,7 @@ const ActivityFeedList: FC<ActivityFeedListProp> = ({
         <Fragment>
           {relativeDays.map((d, i) => {
             return (
-              <Fragment key={i}>
+              <div data-testid="feed" key={i}>
```

Comment:
For more than 1 relative days, we should also add index to the test id

`data-testid="feed"+i`